### PR TITLE
Add Share Button

### PR DESCRIPTION
**Pull request by zugzwang03**
So, the share button has been done with the required features essential. Sadly, I could not make it responsive to all devices, it would be better if some one else fixes that issue. 
Issue number #29 resolved.
The working can be checked from here : https://zugzwang03.github.io/tarana/
clipboard

### CODE REVIEW

**Comment by flyingsonu122** [comment]
@zugzwang03 please look up https://www.sonu.live/diwali-wishes/ 
do like that or more

**Comment by zugzwang03** [comment]
I made some changes, can you review it?
https://zugzwang03.github.io/tarana/

**Comment by flyingsonu122** [comment]
> I made some changes, can you review it? https://zugzwang03.github.io/tarana/

- remove youtube
- link should be like this **https://sonu.live/tarana/play.html?id={ID}**
- make it more responsive to all screen size



@sulogna2001 please work together with @zugzwang03 to make it happen

File: up.js [hunk 1]
```diff
@@ -353,6 +356,7 @@ function out() {
     title.innerHTML = All_song[index_no].name;
     track_image.src = All_song[index_no].img;
     artist.innerHTML = All_song[index_no].singer;
+    links.innerHTML = All_song[index_no].path;
```
Comment (flyingsonu122) [inline up.js:359]: links.innerHTML = `https://sonu.live/tarana/play.html?id=${All_song[index_no].id}`;

File: up.js [hunk 2]
```diff
@@ -79,6 +81,7 @@ function genLink(e) {
         title.innerHTML = All_song[e.id - 1].name;
         track_image.src = e.img;
         artist.innerHTML = e.singer;
+        links.innerHTML = e.path;
```
Comment (flyingsonu122) [inline up.js:84]: image

**Review by flyingsonu122 — CHANGES_REQUESTED** [review]
modify and see is it working

**Comment by zugzwang03** [comment]
Did those changes.

**Review by flyingsonu122 — APPROVED** [review]
Thank you @zugzwang03 for your contributions

**Pull request merged**